hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/MembershipManager.java

```diff
@@ -789,6 +803,11 @@ private void sortMembersInMembershipOrder(List<Member> members) {
 
     private void sendMembershipEventNotifications(MemberImpl member, Set<Member> members, final boolean added) {
         int eventType = added ? MembershipEvent.MEMBER_ADDED : MembershipEvent.MEMBER_REMOVED;
+        node.getNodeExtension().getAuditlogService()
+                .eventBuilder(added ? AuditlogTypeIds.CLUSTER_MEMBER_ADDED : AuditlogTypeIds.CLUSTER_MEMBER_REMOVED)
+                .message("Membership changed")
+                .addParameter("memberAddress", member.getAddress())
```

Comment:
Do you want to log if it was added or removed?

Reply:
Do you mean to have it in the message too? Currently, the add and remove types are differentiated in `typeId()`.

Reply:
Ah, right, missed that, thanks.